testsuite/gateways/apicast/operator.py

```diff
@@ -216,6 +216,12 @@ def _update(apicast):
         self.apicast.modify_and_apply(_update)
         self.reload()
 
+    def disconnect_open_telemetry(self, *args):
+        """
+        Current tests delete apicast after opentelemetry tests, cleanup of the
+        apicast itself will clean all configuration for opentelemetry as well
+        """
```

Comment:
Shouldn't there be at least `pass` ?

Reply:
doc string is a valid content of a empty function by itself
https://pylint.pycqa.org/en/v3.0.3/user_guide/messages/warning/unnecessary-pass.html